### DISCUSSION TITLE
plantFEMの追加の提案

### DISCUSSION THREAD

**Discussion by tkoyama010** [FEABeRjp/FEABeR#33] [Ideas]
植物/農作物シュミレーター[plantFEM](https://github.com/kazulagi/plantFEM)の追加を提案します。
CC @kazulagi

**Comment by mmer547** [comment]
いいと思います

**Comment by mmer547** [comment]
開発者の方に聞いたところ、来月中頃に安定版リリースあるらしく、間に合うならデータ入れてもらえるとのことです
